<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nc_commitPersonInfo</class>
 <widget class="QWidget" name="nc_commitPersonInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EnsureInformation{background-color: #ffffff;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_Photo">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>360</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(124, 124, 124);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>姓  名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Name">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>61</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>穆罕默德.艾罕穆德.买买提</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
             </property>
             <property name="text">
              <string>性别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Gender">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
             </property>
             <property name="text">
              <string>男</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
             </property>
             <property name="text">
              <string>民族</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_Nation">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
             </property>
             <property name="text">
              <string>唯吾尔族</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>出生日期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Birthday">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>2012年10月11日</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 85, 255);font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
             </property>
             <property name="text">
              <string>住址</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_Address">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>61</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>上海市闵行区七宝镇新龙路1333号</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>身份证</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Identity">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>401233201010101034</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_Career">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>61</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>思源黑体 CN Medium</family>
             <pointsize>-1</pointsize>
             <weight>7</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>职业</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Career">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;
}
QComboBox::drop-down {			/*下拉按钮*/
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 36px;
      border:none;
}

QComboBox::down-arrow {		/*下拉按钮*/
 	image: url(:/Image/down_arrow.png);
  }

QComboBox QAbstractItemView{	/*列表 */
	margin-top: 8px;
	margin-left: 6px;
	margin-bottom: 6px;
	margin-right: 6px;
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    outline: 2px;
    padding-left: 16px;
    padding-top: 12px;
	padding-bottom:12px;
  }
  
 QComboBox::QAbstractItemView::item {		/*下拉按钮*/
 	height:56px;
	margin-top: 8px;
	margin-bottom: 8px;
 }

QComboBox QScrollBar::vertical{ /*滑条 主体部分*/
    width:36px;
    background:rgb(255,255,255);
    border:none;
}
QComboBox QScrollBar::handle::vertical{ /*滑块主体*/
    border-radius:3px;
    background-color: rgba(30, 34, 43, 0.2);
}
QComboBox QScrollBar::handle::vertical::hover{
    background-color: rgba(30, 34, 43, 0.2);
}
QComboBox QScrollBar::add-line::vertical{/*上箭头*/
    border:none;
}
QComboBox QScrollBar::sub-line::vertical{/*下箭头*/
    border:none;
}
</string>
           </property>
           <item>
            <property name="text">
             <string>单位负责人</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>专业、技术人员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>办事人员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>商业、服务人员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>农林牧渔水利生产人员</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>生产运输工人</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>其他从业人员</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_Bank">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>61</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>思源黑体 CN Medium</family>
             <pointsize>-1</pointsize>
             <weight>7</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>服务银行</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_BankName">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-weight: bold;
line-height: 44px;
letter-spacing: 1px;
color: #727275;
font: 57 42px &quot;思源黑体 CN Medium&quot;;</string>
           </property>
           <property name="text">
            <string>农商行</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="minimumSize">
        <size>
         <width>266</width>
         <height>101</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color: #298cf0;
border-radius: 25px;
font: 57 42px &quot;思源黑体 CN Medium&quot;;
font-weight: normal;
line-height: 49px;
letter-spacing: 1px;
color: #ffffff;}
QPushButton:pressed{background:#38a357;}
</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
